stats/views.py

```diff
@@ -80,6 +101,94 @@ def summary(request):
     return Response(status=status.HTTP_404_NOT_FOUND)
 
 
+@csrf_exempt
+def sec_cost_centres(request):
+    if request.method == "POST":
```

Comment:
This should be a GET request

Reply:
I need to make a POST as every secondary cost center is dependent on the primary cost center. Maybe there is an easier way to do it with a GET.

Reply:
You can for instance use query parameters to send along data, if that's what you mean